eng/pipelines/stress-test-release.yml

```diff
@@ -76,7 +73,10 @@ jobs:
     - task: AzureCLI@2
       displayName: Build and Deploy Stress Tests
       inputs:
-        azureSubscription: ${{ parameters.Subscription }}
+        ${{ if eq(parameters.Environment, 'prod') }}:
+          azureSubscription: Azure SDK Test Resources
+        ${{ if eq(parameters.Environment, 'test') }}:
+          azureSubscription: Azure SDK Developer Playground
```

Comment:
Should test be playground? Should it be a test sub or should we change this to playground env?

Reply:
I think `test` implies that is a resource used for testing, whereas `prod` implies a resource used for workloads that aren't touched directly and have more stability. `Azure SDK Test Resources` has the word "test" but we treat it as a "production" resource for testing workloads. So I'd be more inclined to keep the subscriptions the same, but call the environments `prod` and `playground` as that's probably clearer to developers using the scripts. @richardpark-msft thoughts?

Reply:
I definitely like renaming "test" to "playground" but I'm still not sure I follow the name "prod", which generally implies to me production vs staging. However I don't care much as long as docs call out what it means... where are the docs? :)

Reply:
Per env docs are located here (anchor link to prod): https://github.com/Azure/azure-sdk-tools/blob/main/tools/stress-cluster/cluster/README.md#prod-cluster

Reply:
For test -> playground, I'd like to do this in a separate phase since we'll want to rotate the cluster as well and spin people's workloads down from the current one. So I'd prefer to merge this as is and then track the rename via https://github.com/Azure/azure-sdk-tools/issues/2493.

Reply:
Sounds good.